src/spatial/models/sentinel2_context_model.py

```
@@ -174,6 +182,11 @@ def _sample_item(self, item, latitude, longitude):
         if any(value is None for value in reflectance.values()):
             raise ValueError("One or more Sentinel-2 reflectance bands had no valid data")
 
+        scl_class = int(round(scl))
```

Ruff (0.15.20)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)